src/components/views/Registration.js

```diff
@@ -47,6 +47,7 @@ const Registration = () => {
   let hundredYearsAgo = new Date()
   hundredYearsAgo.setFullYear(hundredYearsAgo.getFullYear() - 100)
   const [birthday, setBirthday] = useState(eighteenYearsAgo);
+  const [location, setLocation] = useState([0, 0]);
```

Comment:
Acutally, I find this quite clever to set default to 0,0. Because this is the northpole (?), and there isn't anyone anyway, it creates a pool of people that haven't allowed for position. Togehter with default range of 10km, they just interact with each other.

Reply:
yes exactly, that was my thought. only correction, 0,0 is somewhere in the sea in front of the african east coast, but no islands near and apparently for many geo coordinate applications using 0,0 is the equivalent of null, because it would almost certantly not occur by chance